README.md

```diff
@@ -64,19 +64,65 @@ $ balena login
 
 ### Proxy support
 
-HTTP(S) proxies can be configured through any of the following methods, in order of preference:
-
-* Set the `BALENARC_PROXY` environment variable in URL format (with protocol, host, port, and
-  optionally basic auth).
-* Alternatively, use the [balena config file](https://www.npmjs.com/package/balena-settings-client#documentation)
-  (project-specific or user-level) and set the `proxy` setting. It can be:
-  * A string in URL format, or
-  * An object in the [global-tunnel-ng options format](https://www.npmjs.com/package/global-tunnel-ng#options) (which allows more control).
-* Alternatively, set the conventional `https_proxy` / `HTTPS_PROXY` / `http_proxy` / `HTTP_PROXY`
-  environment variable (in the same standard URL format).
-
-To get a proxy to work with the `balena ssh` command, check the
-[installation instructions](https://github.com/balena-io/balena-cli/blob/master/INSTALL.md).
+HTTP(S) proxies can be configured through any of the following methods, in precedence order
+(from higher to lower):
+
+* The `BALENARC_PROXY` environment variable in URL format, with protocol (`http` or `https`),
+  host, port and optionally basic auth. Examples:
+  * `export BALENARC_PROXY='https://bob:secret@proxy.company.com:12345'`
+  * `export BALENARC_PROXY='http://localhost:8000'`
+
+* The `proxy` setting in the [CLI config
+  file](https://www.npmjs.com/package/balena-settings-client#documentation). It may be:
+  * A string in URL format, e.g. `proxy: 'http://localhost:8000'`
+  * An object in the format:
+
+    ```yaml
+    proxy:
+        protocol: 'http'
+        host: 'proxy.company.com'
+        port: 12345
+        proxyAuth: 'bob:secret'
+    ```
+
+* The `HTTPS_PROXY` and/or `HTTP_PROXY` environment variables, in the same URL format as
+  `BALENARC_PROXY`.
+
+> Note: The `balena ssh` command has additional setup requirements to work behind a proxy.  
+> Check the [installation instructions](https://github.com/balena-io/balena-cli/blob/master/INSTALL.md).
+
+Some installations of the balena CLI also include support for the `BALENARC_NO_PROXY` environment
+variable, which allows proxy exclusion patterns to be defined. The current support status is listed
+below. Eventually, all installation types will have support for it.
+
+OS | Installation type | BALENARC_NO_PROXY environment variable support
+-- | ----------------- | ----------------------------------------------
+Windows | standalone zip | Supported with CLI v11.24.0 and later
+Windows | native/GUI     | Not supported
+macOS   | standalone zip | Not supported
+macOS   | native/GUI     | Supported with CLI v11.24.0 and later
+Linux   | standalone zip | Not supported
+Any     | npm            | Supported with Node.js >= v10.16.0 and CLI >= v11.24.0
```

Comment:
This "complex situation" comes from the fact that the proxy exclusion feature is implemented by the `global-agent` package, which only supports Node 10 and later (see more details in `app-common.ts`). Different installation types use / bundle different versions of Node.js, which a balena CI configuration (changes require pull requests to `resin-concourse`, which we will do in due course).